tests/test_log.py

```diff
@@ -1,6 +1,39 @@
+import logging
+
 import pytest
 
-from mandible.log import log_errors
+from mandible.log import init_custom_log_record_factory, log_errors
+
+
+@pytest.fixture
+def event_config():
+    return {
+        "cma": {
+            "event": {
+                "cumulus_meta": {
+                    "execution_name": "this_is_a_test_execution",
+                    "cumulus_version": "TEST_CUMULUS"
+                },
+                "payload": {
+                    "granules": [{"granuleId": "test_granule"}]
+                }
+            }
+        }
+    }
+
+
+def test_custom_log_record_factory(caplog, monkeypatch, event_config):
+    monkeypatch.setenv("DAAC_VERSION", "TEST_DAAC")
+    monkeypatch.setenv("CORE_VERSION", "TEST_CORE")
+    caplog.set_level(logging.INFO)
+    log = logging.getLogger()
+    init_custom_log_record_factory(event_config)
```

Comment:
If we're setting globals here we should make sure to unset them after the test is over. The best way to do that is with a fixture to guarantee that the cleanup code is run regardless of if the test passed or errored out.

Maintaining test isolation is very important to prevent confusing and unpredictable test behavior.